.github/CODEOWNERS

```diff
@@ -340,25 +340,25 @@ lerna.json @grafana/frontend-ops
 # public folder
 /public/app/core/ @grafana/user-essentials
 /public/app/core/components/TimePicker/ @grafana/user-essentials
-/public/app/core/components/Layers/ @grafana/grafana-edge-squad
+/public/app/core/components/Layers/ @grafana/dataviz-squad
 /public/app/features/all.ts @grafana/user-essentials
 /public/app/features/admin/ @grafana/grafana-authnz-team
 /public/app/features/annotations/ @grafana/user-essentials
 /public/app/features/api-keys/ @grafana/user-essentials
-/public/app/features/canvas/ @grafana/grafana-edge-squad
+/public/app/features/canvas/ @grafana/dataviz-squad
 /public/app/features/commandPalette/ @grafana/user-essentials
-/public/app/features/comments/ @grafana/grafana-edge-squad
+/public/app/features/comments/ @grafana/dataviz-squad
```

Comment:
Same comment here - I think this is from an experimental feature Alexander worked on - so not sure we should keep this